scripts/add_snowflake_hive_metastore_connector_script_action.sh

```diff
@@ -22,7 +22,7 @@ SNOWFLAKECONFDIR=/usr/hdp/current/hive-server2/conf/conf.server
 
 echo "Importing helper module"
 # Import helper module
-wget -O /tmp/HDInsightUtilities-v01.sh -q https://hdiconfigactions.blob.core.windows.net/linuxconfigactionmodulev01/HDInsightUtilities-v01.sh && source /tmp/HDInsightUtilities-v01.sh && rm -f /tmp/HDInsightUtilities-v01.sh
+source <(wget -q -O - https://hdiconfigactions.blob.core.windows.net/linuxconfigactionmodulev01/HDInsightUtilities-v01.sh)
```

Comment:
Is this an extra `-` ?

Reply:
It's intentional, it's meant as an argument to `-O` to redirect to stdout IIRC.